src/Symfony/Component/Mailer/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ CHANGELOG
 ---
 
  * Add `logger` (constructor) property to `RoundRobinTransport`
+ * Add `subaccount` parameter to `MandrillApiTransport` payload
```

Comment:
this must be added in the changelog of the mailchimp bridge instead, not in the changelog of `symfony/mailer`